lighthouse/helpers/plates.py

```diff
@@ -529,116 +487,216 @@ def create_cherrypicked_post_body(
+        logger.error("Failed to construct a cherrypicking plate failed message")
+        logger.exception(e)
+        return [
+            "An unexpected error occurred attempting to construct the cherrypicking plate "
+            f"failed event message: {e}"
+        ], None
+
+
+# Private methods
+
+
+def __ss_sample_subjects(samples):
     subjects = []
     for sample in samples:
         if "control" in sample:
             subject = {
                 "role_type": "control",
                 "subject_type": "sample",
-                "friendly_name": control_friendly_name(sample),
+                "friendly_name": __ss_control_friendly_name(sample),
                 "uuid": sample["uuid"],
             }
         else:
             subject = {
                 "role_type": "sample",
                 "subject_type": "sample",
-                "friendly_name": sample_friendly_name(sample),
+                "friendly_name": __ss_sample_friendly_name(sample),
                 "uuid": sample["uuid"],
             }
         subjects.append(subject)
     return subjects
 
 
-def sample_friendly_name(sample):
+def __ss_control_friendly_name(sample):
+    return f"{sample['supplier_name']}"
+
+
+def __ss_sample_friendly_name(sample):
     name = "__".join(
         [sample["sample_description"], sample["name"], sample["lab_id"], sample["result"]]
     )
     return name
 
 
-def control_friendly_name(sample):
-    return f"{sample['supplier_name']}"
-
-
-def source_plate_subjects(plate_id_mappings):
-    subjects = []
-    for mapping in plate_id_mappings:
-        subject = {
-            "role_type": "cherrypicking_source_labware",
-            "subject_type": "plate",
-            "friendly_name": mapping["barcode"],
-            "uuid": mapping["uuid"],
-        }
-        subjects.append(subject)
-    return subjects
-
-
-def robot_subject(robot_serial_number):
-    try:
-        robot_mapping = app.config["BECKMAN_ROBOTS"][robot_serial_number]
-    except KeyError:
-        logger.error("Unable to find events information for robot:" + robot_serial_number)
-        raise
-    try:
-        robot_friendly_name = robot_mapping["name"]
-    except KeyError:
-        logger.error("Unable to find friendly name for robot: " + robot_serial_number)
-        raise
-
-    try:
-        robot_uuid = robot_mapping["uuid"]
-    except KeyError:
-        logger.error("Unable to find UUID for robot: " + robot_serial_number)
-        raise
-
-    subject = {
-        "role_type": "robot",
-        "subject_type": "robot",
-        "friendly_name": robot_friendly_name,
-        "uuid": robot_uuid,
+def __supplier_name_for_dart_control(dart_row):
+    args = {
+        "control_type": dart_row[FIELD_DART_CONTROL],
+        "source_barcode": dart_row[FIELD_DART_SOURCE_BARCODE],
+        "source_coordinate": dart_row[FIELD_DART_SOURCE_COORDINATE],
     }
+    return "{control_type} control: {source_barcode}_{source_coordinate}".format(**args)
```

Comment:
Any benefit over using an f-string?

Reply:
😆 I asked the same of Eduardo when I first saw it: the reason was line length restrictions. If both of us think this is strange, I will change this to use f-strings

Reply:
Yeah, I'm liking the new 120 line length in crawler, seems to make __most__ things more readable but keeping it easy to review in github etc. and side-by-by compare.